src/libs/ReportUtils.ts

```diff
@@ -831,6 +836,40 @@ function isConciergeChatReport(report: OnyxEntry<Report>): boolean {
     return report?.participantAccountIDs?.length === 1 && Number(report.participantAccountIDs?.[0]) === CONST.ACCOUNT_ID.CONCIERGE && !isChatThread(report);
 }
 
+/**
+ * Checks if the supplied report belongs to workspace based on the provided params.
```

Comment:
Could you add an explanation here for what the fake policy ID helps to fetch for example?